src/Paket.Core/LockFile.fs

```diff
@@ -179,6 +179,10 @@ module LockFileSerializer =
                     | None -> ()
                     | Some command -> yield "      build: " + command
 
+                    match file.PackagePath with
+                    | None -> ()
+                    | Some path -> yield "      path: " + path
+
```

Comment:
@forki is it ok to add this info to the lock file? It's being used in `RemoteDownload` module